content/docs/get-started-with-neon/setting-up-a-project.md

```diff
@@ -6,7 +6,7 @@ To set up a Neon project:
 
 1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
 2. If you are creating your very first project, click **Create the first project**. Otherwise, click **New Project**. 
-3. Enter a name for your project and click **Create Project**. Upon creating a project, you are presented with a dialog that provides the following information:
+3. Specify a name, a PostgreSQL version, and click **Create Project**. Upon creating a project, you are presented with a dialog that provides the following information:
```

Comment:
PostgreSQL version is optional.
Should we mention default here or it will be too wordy?